ambry-tools/src/main/java/com.github.ambry/account/AccountUpdateTool.java

```diff
@@ -207,37 +229,9 @@ public static void main(String args[]) throws Exception {
    * @param containerJsonVersion The {@link Container} JSON version to write in.
    * @throws Exception
    */
-  static void updateAccount(String accountJsonFilePath, String zkServer, String storePath, String backupDir,
-      int zkConnectionTimeoutMs, int zkSessionTimeoutMs, short containerJsonVersion) throws Exception {
+  AccountUpdateTool(String zkServer, String storePath, String backupDir, int zkConnectionTimeoutMs,
```

Comment:
suggestion: if we want to use this as a library, it _may_ make sense to pass the `accountService` in instead of creating it within the constructor

Reply:
I think the class also works with `AccountService` (as opposed to `HelixAccountService` only)

Reply:
that's a good idea, extracted the AccountService construction code into a helper method.